plugin.php

```diff
@@ -306,7 +306,7 @@ function rest_validate_request_arg( $value, $request, $param ) {
 		if ( isset( $args['format'] ) ) {
 			switch ( $args['format'] ) {
 				case 'date-time' :
-					if ( ! rest_parse_date( $value ) ) {
+					if ( $value && ! rest_parse_date( $value ) ) {
```

Comment:
Thanks for the feedback: note that what I am trying to do is update a post that is new, and unpublished. I've tested with posts created in wp-admin or via the api. new, unpublished posts have '0000-00-00 00:00:00' for their gmt_date and blank slugs. Updates to the endpoint as an update with these values fails validation. 

perhaps the api should just not return a slug and gmt_date for new unpublished posts? i'm going to try adjusting the client to skip sending these values to see if that resolves my issue.